src/modules/auth/index.js

```diff
@@ -54,6 +56,20 @@ export default class SeaCatAuthModule extends Module {
 			component: ResourcesCreateContainer,
 			resource: "seacat:resource:access"
 		});
+		app.Router.addRoute({
+			path: '/auth/resources-deleted',
+			exact: true,
+			name: 'Deleted resources',
+			component: ResourcesDeletedListContainer,
+			resource: 'seacat:resource:access'
```

Comment:
@petrKavulok Clarify with @byewokko if this screen shouldnt be accessed only by superuser

Reply:
it is

Reply:
@petrKavulok Then also other actions regarding hard deleting resource(s) will be only for superusers I guess (and not `seacat:resource:edit`)